src/StreamingDataFrames/streamingdataframes/models/serializers/quix.py

```diff
@@ -407,3 +418,8 @@ def __call__(
         }
 
         return self._to_json(result)
+
+
+class QuixLegacyEventsSerializer(QuixEventsSerializer, QuixLegacySerializer):
```

Comment:
If we still deserialize legacy and new format with the same class, could we maybe do the same with serialization?

E.g. we could have one `QuixEventsSerializer` class with the flag `legacy: bool` in the constructor and go from there. 
Same for `QuixTimeseriesSerializer`.

Reply:
Changed